cgroup_top.py

```diff
@@ -49,6 +50,38 @@ def cmd_exists(cmd):
     except OSError:
         return False
 
+
+def docker_container_name(container_id, default, cache=dict()):
```

Comment:
This is a clever way to do caching but few people know that the "default" value is shared between calls. Can you add a comment to hint them?